Analysis/Tutorials/src/configurableObjects.cxx

```diff
@@ -78,7 +78,10 @@ struct ConfigurableObjectDemo {
 
   void process(aod::Collision const&, aod::Tracks const& tracks)
   {
-    LOGF(INFO, "Cut1: %.3f; Cut2: %.3f", cut, mutable_cut);
+    std::stringstream tmpcut, tmpmutable_cut;
+    tmpcut << cut;
+    tmpmutable_cut << mutable_cut;
+    LOGF(INFO, "Cut1: %s; Cut2: %s", tmpcut.str(), tmpmutable_cut.str());
```

Comment:
OK, updated the PR. Here the `(float)` trick didn't work, so I stick with `stringstream` but changed `%f%` to `%s`